src/components/screens/monitor/delegates/overview.js

```diff
@@ -44,7 +44,7 @@ const Overview = ({
             typeof chartActiveAndStandby.data === 'number'
               ? (
                 <div className={styles.chartBox}>
-                  <h1 className={styles.chartTitle}>{t('Delegates')}</h1>
+                  <h2 className={styles.chartTitle}>{t('Delegates')}</h2>
```

Comment:
- 💡 Semantically, this should already be
```suggestion
                  <h3 className={styles.chartTitle}>{t('Delegates')}</h3>
```
and above
```
<h1>{t('Overview')}</h1>
```
should be
```
<h2>{t('Overview')}</h2>
```
Because the `<h1>` of this page is `Monitor`, outside this file

Reply:
Unfortunately we don't follow semantic HTML in this project. I changed this tag to reuse an existing style, if I change it to h3, then I have to touch other components out of scope of this PR. Definitely a good idea to do so, but let's do it in its dedicated issue.